action.yml

```diff
@@ -11,8 +11,12 @@ inputs:
     description: "The channel, or a specific version from a channel to install ('stable', 'beta', 'dev', '2.7.2', '2.12.0-1.4.beta'). Using one of the three channels instead of a version will give you the latest version published to that channel."
     required: false
     default: "stable"
+  architecture:
+    description: "The CPU architecture ('x64 or ia32'). Supported on macOS and Linux only."
```

Comment:
Why not Windows?

Reply:
In fact, the test is on windows...

Reply:
Hmm, not my a-game when I created this PR. We have ia32 for Windows and Linux, but not macOS, right?

Reply:
windows: ia32, x64, (we should probably add arm64 to that list at some point but let's leave that to the Dart PM).
linux: ia32, x64, arm, arm64
mac: x64, (arm64 when we support Apple Silicon natively).